app/components/chat/input-message.ts

```diff
@@ -22,6 +22,35 @@ export default class ChatInputMessage extends Component {
     return this.inputMessage.length > 0;
   }
 
+  async getBase64Data(blob: Blob): Promise<any> {
+    return new Promise((resolve, reject) => {
+      var reader = new FileReader();
+      reader.onloadend = () => {
+        resolve(reader.result);
+      }
+      reader.onerror = (err: ProgressEvent) => {
+        reject(err)
+      }
+      reader.readAsDataURL(blob);
+    });
+  }
+
+  @action
+  async onPasteInput() {
+    var items = await navigator.clipboard.read();
```

Comment:
```suggestion
  const items = typeof navigator?.clipboard?.read === 'function' ? await navigator.clipboard.read() : e.clipboardData.files;
```

Haven't tested this but found this fallback to the old clipboard API here https://web.dev/patterns/clipboard/paste-images/

Reply:
this wouldn't work in firefox so we should save this for when they update it